libraries/AP_HAL_ChibiOS/hwdef/CubeNode-vehicle/readme.md

```diff
@@ -0,0 +1,115 @@
+# CubeNode-vehicle
+
+This build can be used to flash a vehicle code on a CubeNode. This will have to be loaded using dfu as the CubeNode ships with a Periph bootloader which cannot load a vehicle binary.
+
+This can be used as a starting point for other configurations, pin functions can be add, removed, or changed as needed.
+
+## Serial
+
+Three serial physical ports are available along with two USB endpoints. The ordering is:
+
+#### SERIAL0
+
+First USB endpoint
+
+#### SERIAL1
+
+| Pin name | Pin number | function |
+| -------- | ---------- | -------- |
+| PE1      | 3          | UART8_TX |
+| PE0      | 4          | UART8_RX |
+
+#### SERIAL2
+
+| Pin name | Pin number | function  |
+| -------- | ---------- | --------- |
+| PC6      | 44         | USART6_TX |
+| PC7      | 43         | USART6_RX |
+
+#### SERIAL3
+
+UART 3 has flow control.
+
+| Pin name | Pin number | function  |
+| -------- | ---------- | --------- |
+| PE7      | 23         | UART7_RX  |
+| PE8      | 22         | UART7_TX  |
+| PE10     | 21         | UART7_CTS |
+| PE9      | 72         | UART7_RTS |
+
+##### SERIAL4
+
+Second USB endpoint
+
+# CAN
+
+Two CAN busses are available, transceivers are within the Node itself but termination resistors are not.
+
+#### CAN 1
+
+| Pin name | Pin number | function  |
+| -------- | ---------- | --------- |
+| CAN1_H   | 2          | CAN1 high |
+| CAN1_L   | 1          | CAN1 low  |
+
+#### CAN 2
+
+| Pin name | Pin number | function  |
+| -------- | ---------- | --------- |
+| CAN2_H   | 15         | CAN2 high |
+| CAN2_L   | 14         | CAN2 low  |
+
+# I2C
+
+A single I2C bus is provided, software pullup resistors are enabled.
```

Comment:
Of what value? The ones in the microcontroller are not spec-compliant.

Reply:
There just the microcontroller ones, seem to work fine. I could add a not that there out of spec I guess. (quite a few other boards do it this way too).

Reply:
Would appreciate such a note.

Context: https://github.com/ArduPilot/ardupilot/pull/31070#issuecomment-3315124896